### ISSUE TITLE
Putaminal eye sign

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11391]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 40821445 Brain magnetic resonance imaging (MRI) showed bilateral symmetrical T2 signal hyperintensities in basal ganglia involving mainly the lentiform nucleus with mid-dorsal putamen sparing (**putaminal eye sign**) in association with mild periventricular deep white matter hyperintensities and brain atrophic changes.

PMID: 37996139 The middorsal putamen was spared bilaterally. 
The characteristic sparing of the middorsal putamen giving an eye appearance is pathognomonic for 3-MEthylGlutaconic aciduria, Deafness, Encephalopathy, Leigh (MEGDEL)- like syndrome and allowed diagnosis based on MRI findings.
MEGDEL syndrome is an autosomal recessive disease impairing mitochondrial function. 
This rare disorder is caused by mutations in the SERAC1 gene, which encodes for mitochondrial membrane protein. MEGDEL syndrome is a 
disorder with infantile onset which is initially misdiagnosed as Leigh syndrome. All patients with suspected disease had a distinctive MRI 
brain pattern affecting the basal ganglia, particularly putamen. Individuals with MEGDEL syndrome have a unique MRI pattern with five 
distinct disease stages. The first stage involves T2 signal changes involving globus pallidus. The second stage has swelling of the putamen and 
caudate nucleus. The dorsal putamen is unaffected, and this normal area surrounded by T2 hyperintensity is referred to as putaminal eye. From the third stage onwards, there is a gradual decrease of the ‘**putaminal eye**’. The fourth and fifth stages were characterised by atrophy of basal ganglia. The basal ganglia dysfunction increases with increasing stages.

PMID: 24741715 Bilateral basal ganglia involvement is seen on brain MRI (comparable to Leigh syndrome) [Wortmann et al 
2015]. All affected individuals with MEGD(H)EL syndrome had a distinctive brain MRI pattern with five
characteristic disease stages affecting the basal ganglia, especially the putamen.
• Stage 0. Normal MRI
• Stage 1. T2-weighted signal changes present in the pallidum
• Stage 2. Swelling of the putamen and caudate nucleus. The dorsal putamen contains an "eye" that shows no 
signal alteration and (thus) seems to be spared during this stage of the disease.
• Stage 3. The **putaminal eye** increases, reflecting progressive putaminal involvement. This "eye" was found 
in all individuals with MEGD(H)EL syndrome during a specific age range (>1-4 years), and has not been 
reported in other disorders, making it pathognomonic for MEGD(H)EL syndrome and allowing diagnosis 
based on MRI findings.
• Stage 4. Basal ganglia degeneration until near loss; cortical and cerebellar atrophy

(https://radiopaedia.org/articles/megdel-syndrome?lang=us)

**Parent term (use hpo.jax.org/app)**
Abnormal basal ganglia morphology [HP:0002134]

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MEGDEL syndrome ORPHA:352328

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001454
- label: Putaminal eye sign
- parents: [Abnormal putamen morphology (HP:0031982)](https://hpo.jax.org/app/browse/term/HP:0031982)
- pmid: PMID:25642805|PMID:37996139
- def: An imaging anomly of the putamen that is thought to be pathognomic for MEGDEL syndrome (3-MethylGlutaconic aciduria, Deafness, Encephalopathy, Leigh-like syndrome MIM #614739). In stage 1 of the disease, T2 signal changes of the pallidum are present. In stage 2, swelling of the putamen and caudate nucleus is seen. The dorsal putamen contains an eye that shows no signal alteration and (thus) seems to be spared during this stage of the disease. It later increases, reflecting progressive putaminal involvement. Sparing of the mid dorsal putamen (showing normal signal surrounded by T2 hyperintense signal) is referred to as the putaminal eye.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11391
- synonyms:
